packages/mui-material/src/Button/buttonClasses.ts

```diff
@@ -176,6 +176,22 @@ export interface ButtonClasses {
   colorInfo: string;
   /** Styles applied to the root element if `color="warning"`. */
   colorWarning: string;
+  /** Styles applied to the span element that wraps the children when `loading={true}`. */
+  label: string;
```

Comment:
This is confusing. Ideally the name `label` should clearly state what's it all about and one needs to read the class description to know that it's only applied when it's in the loading state.